gradle-plugin/src/main/kotlin/dev/icerock/gradle/generator/ColorsGenerator.kt

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.gradle.generator
+
+import com.android.utils.forEach
+import com.squareup.kotlinpoet.ClassName
+import com.squareup.kotlinpoet.CodeBlock
+import com.squareup.kotlinpoet.KModifier
+import com.squareup.kotlinpoet.PropertySpec
+import com.squareup.kotlinpoet.TypeSpec
+import dev.icerock.gradle.generator.android.AndroidColorsGenerator
+import dev.icerock.gradle.generator.common.CommonColorsGenerator
+import dev.icerock.gradle.generator.ios.IosColorsGenerator
+import org.gradle.api.file.FileTree
+import java.io.File
+import javax.xml.parsers.DocumentBuilderFactory
+
+abstract class ColorsGenerator(
+    private val colorsFileTree: FileTree
+) : MRGenerator.Generator, ObjectBodyExtendable {
```

Comment:
maybe just move `ObjectBodyExtendable` into `MRGenerator.Generator`? it duplicated in each generator class...

Reply:
Added `ObjectBodyExtendable` inheritance to `MRGenerator.Generator` interface.